tools/apiview/emitters/typespec-apiview/test/apiview.test.ts

```diff
@@ -33,467 +33,580 @@ describe("apiview: tests", () => {
     }
   }
 
-  it("describes model", async () => {
```

Comment:
Here all I did was refactor the tests into smaller pieces and add the test for the enum with spread. Sorry for the big diff 😅